client/src/App.js

```diff
@@ -49,14 +49,41 @@ export default () => {
     signupDisclosure.onClose();
 
     // TODO: Make fetch() request here to signup
-
-    loginDisclosure.onOpen();
+    fetch('http://localhost:5000/signup', {
+      method: 'POST',
+      mode: 'cors',
+      headers: {
+        'Content-type': 'application/json'
+      },
+      body: JSON.stringify(values)
+    })
+    .then(function (data) {
+      console.log('Request succeeded with JSON response', data);
```

Comment:
Might wanna check the data before assuming it's good!  You could always have `data.error`. 😄

Reply:
Yeah good catch- these fetch calls were placeholders while I tried figuring out what the hell was going on.